/**:
  ros__parameters:
    default_position_z: 0.0
    default_velocity_z: 0.0
    default_acceleration_z: 0.0
    default_size_x: 5.0
    default_size_y: 2.0
    default_size_z: 2.0

    # classification remapping from RADAR to Perception
    # Format: [<RADAR label>, <Perception label>]
    # Need to check autoware_sensing_msgs::msg::RadarClassification for RADAR labels
    # and autoware_perception_msgs::msg::ObjectClassification for Perception labels
    classification_remap: [
      "UNKNOWN", "UNKNOWN",
      "CAR", "CAR",
      "TRUCK", "TRUCK",
      "MOTORCYCLE", "CAR", # temporary remap MOTORCYCLE to CAR due to radar tend to misclassify far cars as motorcycles
      "BICYCLE", "CAR",    # temporary remap BICYCLE to CAR due to radar tend to misclassify far cars as bicycles
      "PEDESTRIAN", "PEDESTRIAN",
      "ANIMAL", "ANIMAL",
      "HAZARD", "UNKNOWN"
    ]
